plugin/storage/integration/cassandra_test.go

```diff
@@ -62,6 +62,9 @@ func (*CassandraStorageIntegration) initializeCassandraFactory(t *testing.T, fla
 
 func (s *CassandraStorageIntegration) initializeCassandra(t *testing.T) {
 	f := s.initializeCassandraFactory(t, []string{
+		"--cassandra.basic.allowed-authenticators=",
+		"--cassandra.password=password",
+		"--cassandra.username=username",
```

Comment:
why does this work? Are these the default values for username/password in Cassandra image

Reply:
Ah, I think it's because we have no auth configured on the server, so the extra auth client might send is just getting ignored.